requirements.txt

```diff
@@ -5,6 +5,9 @@ numpy
 # for back_translation
 torch
 
+# for MRValueReplacement
+torchtext==0.9.1
+
```

Comment:
Thanks for your changes @msobrevillac! Let this be added in the requirements.txt of your transformation folder.
Also, happy to see a transformation for key-value pairs!

Reply:
Update done!